compiler/test/fail_compilation/fail1900.d

```diff
@@ -30,7 +30,9 @@ void test1900a()
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/fail1900.d(42): Error: template `imports.fail1900b.Bar(short n)` at fail_compilation/imports/fail1900b.d(2) conflicts with template `imports.fail1900a.Bar(int n)` at fail_compilation/imports/fail1900a.d(2)
+fail_compilation/fail1900.d(44): Error: `Bar` matches conflicting symbols:
```

Comment:
Showing argument type would improve this further

Reply:
Yes, though that would require passing more info to the function.